src/Illuminate/Routing/UrlGenerator.php

```diff
@@ -60,8 +60,9 @@ class UrlGenerator implements UrlGeneratorContract
     protected $cachedRoot;
 
     /**
-     * A cached copy of the URL schema for the current request.
+     * A cached copy of the URL scheme for the current request.
      *
+     * @deprecated In 5.8, this will change to $cachedScheme
```

Comment:
If it's deprecated, you need something to supersede it

Reply:
If the name were `private` it would not constitute a minor BC change.  I don't actually think people are adding macros to UrlGenerator or extending the class in a way where they are directly accessing the cached/in-memory value of the calculated scheme, but to do our due diligence I think we should push the rename to 5.8, just in case.

In the mean time, there is nothing that can simply supersede the original protected property. All it does is track some state, and introducing a secondary (newly named) property would unnecessarily complicate the code for no real value.